src/components/PdfDocument/index.js

```diff
@@ -63,62 +66,156 @@ const BackCoverSection = ({ index, setIndex }) => {
     [index, setIndex]
   );
 
-  return (
-    <Text render={handleRender} />
-  );
+  return <Text render={handleRender} />;
 };
 
 // 需要使用指定Component组织内容，更多可见 https://react-pdf.org/components
 const PDFDocument = ({ performanceData }) => {
+  const [powerIndex, setPowerIndex] = useState(0);
+  const [launchTimeIndex, setLaunchTimeIndex] = useState(0);
+  const [pageLoadTimeIndex, setPageLoadTimeIndex] = useState(0);
+  const [memoryIndex, setMemoryIndex] = useState(0);
+  const [endIndex, setEndIndex] = useState(0);
+
+  // refer to the key of CONTENT_ITEMS
+  const DIRECTORIES = {
+    ios: [
+      "overView",
+      "anr",
+      "powerUsage",
+      "launchTime",
+      "pageLoadTime",
+      "memoryLeak",
+      "about",
+    ],
+    android: [],
```

Comment:
android part is same to ios

Reply:
need include ticket number(eg: MOB-xxx) in commit message.

Reply:
updated